packages/server/tests/acceptance.spec.ts

```diff
@@ -209,6 +214,68 @@ describe('RPC Server Integration Tests', async function () {
         expect(blockResult.transactions.length).to.be.greaterThan(0);
     });
 
+    it('should execute "eth_getBalance" for primary account', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorPrimaryAccount.evm_address, 'latest']);
+        expect(res.data.result).to.not.be.equal('0x0');
+        expect(res.data.result).to.eq('0x566527b339630a400');
+    });
+
+    it('should execute "eth_getBalance" for secondary account', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorSecondaryAccount.evm_address, 'latest']);
+        expect(res.data.result).to.not.be.equal('0x0');
+        expect(res.data.result).to.eq('0x566527b339630a400');
+    });
+
+    it('should execute "eth_getBalance" for non-existing address', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [nonExistingAddress, 'latest']);
+        expect(res.data.result).to.eq('0x0');
+    });
+
+    it('should execute "eth_getBalance" for contract', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorContract.evm_address, 'latest']);
+        expect(res.data.result).to.eq('0x56bc75e2d63100000');
+    });
+
+    it('should execute "eth_getBalance" for account with id converted to evm_address', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [idToEvmAddress(mirrorPrimaryAccount.account), 'latest']);
+        expect(res.data.result).to.eq('0x566527b339630a400');
+    });
+
+    it('should execute "eth_getBalance" for contract with id converted to evm_address', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [idToEvmAddress(contractId.toString()), 'latest']);
+        expect(res.data.result).to.eq('0x56bc75e2d63100000');
+    });
+
+    it('should execute "eth_getTransactionCount" primary', async function () {
```

Comment:
Lets add a test in which we test accounts that do have transactions executed, thus the nonce is not `0x0`

Reply:
This is blocked by the issue described here: https://github.com/hashgraph/hedera-json-rpc-relay/issues/164